tests/test_global_accelerator.py

```diff
@@ -0,0 +1,279 @@
+                        },
+                    },
+                ],
+            },
+            session_factory=session_factory,
+        )
+        p.run()
+
+        result = client.describe_endpoint_group(
+            EndpointGroupArn=endpoint_group_arn)['EndpointGroup']
+        self.assertEqual(result['EndpointGroupArn'], endpoint_group_arn)
+        self.assertEqual(result['ThresholdCount'], 4)
+        endpoint_descriptions = result['EndpointDescriptions']
+        self.assertEqual(endpoint_descriptions[0]['EndpointId'], endpoint_id)
+        self.assertEqual(endpoint_descriptions[0]['Weight'], 0)
+
+    def test_delete_endpoint_group(self):
+
+        session_factory = self.replay_flight_data('test_delete_endpoint_groups_1')
+        client = session_factory().client('globalaccelerator', region_name=available_ga_region)
+        listener_arn = 'arn:aws:globalaccelerator::644160558196:accelerator/' \
+            '1460ad64-e386-41e4-9715-d8bcc665963e/listener/56de20d3'
+
+        result = client.list_endpoint_groups(ListenerArn=listener_arn)
+        self.assertEqual(len(result['EndpointGroups']), 2)
+
+        p = self.load_policy(
+            {
+                'name': 'delete-accelerator-endpoint-group',
+                'resource': 'global-accelerator-endpoint-group',
+                'filters': [{'EndpointGroupRegion': 'us-east-2'}],
+                'actions': [{'type': 'delete'}],
+            },
+            session_factory=session_factory,
+        )
+        p.run()
+
+        session_factory = self.replay_flight_data('test_delete_endpoint_groups_2')
+        client = session_factory().client('globalaccelerator', region_name=available_ga_region)
+        result = client.list_endpoint_groups(ListenerArn=listener_arn)
+        self.assertEqual(len(result['EndpointGroups']), 1)
+
+
+class TestListener(BaseTest):
+
+    def test_delete_listener(self):
+        accelerator_arn = 'arn:aws:globalaccelerator::644160558196:accelerator/' \
+            '1460ad64-e386-41e4-9715-d8bcc665963e'
+        session_factory = self.replay_flight_data('test_delete_listeners1')
+        client = session_factory().client('globalaccelerator', region_name=available_ga_region)
+        accelerator_arn = 'arn:aws:globalaccelerator::644160558196:accelerator/' \
+            '1460ad64-e386-41e4-9715-d8bcc665963e'
+        result = client.list_listeners(AcceleratorArn=accelerator_arn)
+        self.assertEqual(len(result['Listeners']), 2)
+
+        p = self.load_policy(
+            {
+                'name': 'delete-accelerator-listener',
+                'resource': 'global-accelerator-listener',
+                'filters': [{'Protocol': 'UDP'}],
+                'actions': [{'type': 'delete'}],
+            },
+            session_factory=session_factory,
+        )
+        p.run()
+
+        session_factory = self.replay_flight_data('test_delete_listeners_2')
```

Comment:
why are these tests using separate flight recordings for their verification, these should be logically part of the same recording.